thirdeye/thirdeye-frontend/app/pods/components/rootcause-header/component.js

```diff
@@ -0,0 +1,33 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  sessionId: null, // 0
+
+  sessionName: null, // ""
+
+  sessionText: null, // ""
+
+  sessionModified: null, // true
+
+  onSave: null, // func(sessionName, sessionText)
+
+  onCopy: null, // func(sessionName, sessionText)
+
+  actions: {
+    onSave() {
+      const { onSave } = this.getProperties('onSave');
```

Comment:
Since you're only getting 1 property, you can just use this.get('onSave')

Reply:
ack. I try to use "getProperties()" consistently rather than mixing up get() and getProperties()